modules/fsi/doc/content/modules/fsi/sqa/fsi_srs.md

```diff
@@ -22,7 +22,7 @@ mechanics module may be used for the solid domain.
 The {{module}} module is designed with the fewest possible constraints on hardware and software.
 For more context on this point, the {{module}} module SRS defers to the framework
 [framework_srs.md#assumptions-and-dependencies] and [Navier-Stokes](navier_stokes_srs.md#assumptions-and-dependencies)
-and [tensor mechanics](tensor_mechanics_srs.md#assumptions-and-dependencies) module assumptions and
+and [solid mechanics](solid_mechanics_srs.md#assumptions-and-dependencies) module assumptions and
```

Comment:
same, cannot

Reply:
The point of all these comments I made was that it's just going to confuse people if you go back and rewrite history. I would lean on the side of keeping the text as it was and getting rid of the hyperlinks, because I don't think they're all that important. I don't really care that much, though, so we can just disagree.